src/main/java/org/jetbrains/bsp/bazel/server/utils/ParsingUtils.java

```diff
@@ -36,16 +36,24 @@ public static URI parseUri(String uri) {
     }
   }
 
-  // TODO FOR REVIEW: naming suggestions? what exactly is the thing that it returns?
-  public static List<String> parseClasspathFromFile(URI protoPathUri) {
+  public static StatusCode parseExitCode(int exitCode) {
```

Comment:
Not sure this is the best place for this to leave (I'm always that fond of utils folders tbh) but it's something to think of another time